apps/judicial-system/web/src/routes/Court/Confirmation/Confirmation.tsx

```diff
@@ -89,20 +90,26 @@ const SigningModal: React.FC<SigningModalProps> = ({
       resSignatureConfirmationResponse: SignatureConfirmationResponse,
     ) => {
       if (resSignatureConfirmationResponse.documentSigned) {
-        const caseCompleted =
-          workingCase.state === CaseState.RECEIVED
-            ? await transitionCase(
-                workingCase,
-                workingCase.decision === CaseDecision.REJECTING
-                  ? CaseTransition.REJECT
-                  : CaseTransition.ACCEPT,
-                setWorkingCase,
-              )
-            : workingCase.state === CaseState.REJECTED ||
-              workingCase.state === CaseState.ACCEPTED
+        try {
+          const caseCompleted =
+            workingCase.state === CaseState.RECEIVED
+              ? await transitionCase(
+                  workingCase,
+                  workingCase.decision === CaseDecision.REJECTING
+                    ? CaseTransition.REJECT
+                    : CaseTransition.ACCEPT,
+                  setWorkingCase,
+                )
+              : workingCase.state === CaseState.REJECTED ||
+                workingCase.state === CaseState.ACCEPTED
 
-        if (caseCompleted) {
-          await sendNotification(workingCase.id, NotificationType.RULING)
+          if (caseCompleted) {
+            await sendNotification(workingCase.id, NotificationType.RULING)
+          } else {
+            // TODO: Handle error
```

Comment:
Should we create a ticket in Asana for these TODOs?

Reply:
I think there already is a ticket for client side error handling. At least it has been discussed multiple times.
There is a general lack of api call error handling in the client.